### PR TITLE
Ticket 8, 9, 10 

### PR DESCRIPTION
**Pull request by CocoS26**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -65,6 +64,34 @@ describe('GET / api/reviews', () => {
...
+});
+
+test("3. 404: send a non-existent column to sort by", () => {
+  return request(app)
+    .get('/api/reviews?sort_by=vote; DROPTABLES')
+    .expect(404)
+    .then(({body:{msg}})=>{
+        expect(msg).toBe('No sort_by found');
+    });
+});
```
Comment (Dutton1990) [inline __tests__/app.test.js:86]: The above would be a 400 as the sort by is completely invalid even though it is the correct data type

File: __tests__/app.test.js [hunk 2]
```diff
@@ -65,6 +64,34 @@ describe('GET / api/reviews', () => {
...
+test("4. 200: accept category query", () => {
+  return request(app)
+    .get('/api/reviews?category=euro game')
+    .expect(200)
+      .then(({body:{reviews}})=>{
+      expect(reviews).toHaveLength(1);
+        })
+});
```
Comment (Dutton1990) [inline __tests__/app.test.js:94]: You can add more to this test, we can check if each review has the category given within it

**Comment by Dutton1990** [comment]
Ticket 8 - PATCH 
Missing a 400 test for invalid id - api/reviews/banana etc. 

Ticket 10 - Queries 
There's quite a few tests you can add for this one (there's lots we can do 😅 ) 
200 - accepts a valid sort by query
200 - accepts a valid order query
400 - invalid order query
404 - non existent category
200 - valid category query but no reviews attached to it, have a look if any of the categories that exist have no reviews attached to them yet 😄 

When you've done the above please re-send the PR to be checked. Great work overall! 🚀

**Review by leekli — APPROVED** [review]
Well done Coco, you are free to merge these branches.

But please do remember to only put 1 ticket on 1 branch at a time, it can get quite difficult for us to keep track of changes if we have multiple tickets in 1 pull request, please nchelp if you're not sure what to do with that.

**Pull request merged**